src/compile/selection/transforms/translate.ts

```diff
@@ -65,30 +62,29 @@ const translate:TransformCompiler = {
 export {translate as default};
 
 function getSign(selCmpt: SelectionComponent, channel: Channel) {
-  let s = channel === X ? '+' : '-';
   if (scalesCompiler.has(selCmpt)) {
-    s = s === '+' ? '-' : '+';
+    return channel === Y ? '+' : '-';
   }
-  return s;
+  return '+';
 }
 
 function onDelta(model: UnitModel, selCmpt: SelectionComponent, channel: Channel, size: 'width' | 'height', signals: any[]) {
   const name = selCmpt.name,
-      signal:any = signals.filter((s:any) => s.name === channelSignalName(selCmpt, channel))[0],
+      hasScales = scalesCompiler.has(selCmpt),
```

Comment:
Don't worry about changing, but just in case you write new code after this - we normally declare one `const` per variable like

```
const a=1;
const b=2;
```

so that the diff for removing one variable only involve one line.